packages/block-library/src/latest-posts/edit.js

```diff
@@ -203,6 +282,7 @@ export default withSelect( ( select, props ) => {
 		order,
 		orderby: orderBy,
 		per_page: postsToShow,
+		_embed: 1,
```

Comment:
Out of curiosity, why `_embed` is prefixed with an underscore? Was it intended to be private?

Reply:
Honestly I have no idea. I found it like this and was wondering about this too

Reply:
_I think_ it is a rest/hateoas/hal convention to have parts of the response which are not part of the normal endpoint data prefixed with underscore (_links, _embedded) so agent enumerating props can skip these easily.